test/cypress/integration/widgets/deployments_view_spec.ts

```diff
@@ -1024,7 +1010,7 @@ describe('Deployments View widget', () => {
             getDeploymentsViewMap().should('not.exist');
         });
 
-        it('should render markers for various deployment states', () => {
+        it.only('should render markers for various deployment states', () => {
```

Comment:
Possible leftover from development phase 🙂 

```suggestion
        it('should render markers for various deployment states', () => {
```